README.md

```diff
@@ -188,6 +188,10 @@ See the notes below.
     This will provide the information about the JDK Maven is using, in case
     it is the wrong one.
 
+    Note that on Ubuntu Focal, the default Maven version is not compatible with
+    newer versions of the JDK; for K's Maven build to work, please ensure you
+    have version 11 installed.
+
```

Comment:
11-17 are fine to use. v18 is breaking the nailgun server.

Reply:
17 breaks maven 3.7, which is shipped only on Focal.